.gitignore

```diff
@@ -299,3 +299,7 @@ Gemfile.lock
 _site/
 
 .jekyll-cache/
+duckdb_functions_join_docs_with_notes.tab.txt
```

Comment:
Did you mean to do this?

Reply:
Yes, there are a few files that are generated when I parsed the existing docs markdown for our scalar functions. It would be good to have them in the gitignore once we have that parsing set up.